python/sglang/srt/utils.py

```diff
@@ -403,7 +403,11 @@ def get_available_gpu_memory(
 
         if empty_cache:
             torch.cuda.empty_cache()
-        free_gpu_memory, _ = torch.cuda.mem_get_info(gpu_id)
+        shared_sysmem_device_mem_sms = (87, 110, 121)  # Orin, Thor, Spark
```

Comment:
This condition is too hacky and can be forward-incompatible.

Reply:
only happens in that devices as Unified memory